lessons/programming/python/file-handling/index.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: File Handling
+---
+
+<CodeLabLayout title="PythonFileHandling">
+
+# File Handling
+Python has built-in functions to perform operations on files, including reading and writing files.
+
+To perform a file operation, first, a file has to be opened. Then the file operation can be performed. Files are closed afterwards.
+
+## Opening a file
+The built-in **open()** function opens and gaining access to a file. It accesses a file by the specified filename (in the current directory) or a path to that file. 
```

Comment:
The built-in **open()** function opens and ~~gaining~~ -> **gains** access to a file.